tests/test_main.py

```diff
@@ -12,7 +12,7 @@
 from app.players.elves.druid import Druid
 from app.players.elves.elf import Elf
 from app.players.elves.elf_ranger import ElfRanger
-from app.players.player import Player
+from app.players.dwarves.dwarf import Player
```

Comment:
The import statement for `Player` from `app.players.dwarves.dwarf` seems incorrect. It should be imported from `app.players.player` instead.